niceman/resource/base.py

```diff
@@ -291,8 +292,7 @@ def __repr__(self):
         return 'Resource({})'.format(self.name)
 
     def add_command(self, command, env=None):
-        """
-        Add a command to the command buffer so that all commands can be
+        """Add a command to the command buffer so that all commands can be
```

Comment:
oh man, I will have so much fun merging ;-)
I know that I somewhat sent you down the rabbit hole with my comments on generic API which required changes only loosely related  to the PR, but these are just stylistic changes better to be avoided when multiple folks known to work on the same pieces (just a note for future)

Reply:
For the merging, you are referring to PR #98? We should probably work on that together in TX